crates/test-utils/src/deploy_helpers.rs

```diff
@@ -634,29 +634,33 @@ impl DeployedKakarot {
     }
 }
 
+/// Returns the dumped Katana state with deployed Kakarot + EVM contracts.
+pub fn dumped_katana_state() -> SerializableState {
```

Comment:
same as before, should this be a method, implemented on SerializableState?
there is no `SerializableState::from_path()` method?
can we implement one?

Reply:
(in a separate issue)

Reply:
`SerializableState` is coming from Katana and doesn't implement this unfortunately, we would need a wrapper if ever